ui/appui-react/src/appui-react/widget-panels/ToolSettings.tsx

```diff
@@ -30,12 +30,20 @@ export interface ToolSettingsEntry {
 }
 
 function EmptyToolSettingsEntry(): ToolSettingsEntry {
-  const labelString = IModelApp.localization.getLocalizedString(
-    "UiFramework:tools.noToolSettings"
+  const toolId = UiFramework.frontstages.activeToolId;
+  const toolName = IModelApp.tools.find(toolId)?.flyover;
+  const toolPlaceholderName = IModelApp.localization.getLocalizedString(
+    "UiFramework:tools.noToolSettingsPlaceholderName"
+  );
+  const labelStringStart = IModelApp.localization.getLocalizedString(
+    "UiFramework:tools.noToolSettingsStart"
+  );
+  const labelStringEnd = IModelApp.localization.getLocalizedString(
+    "UiFramework:tools.noToolSettingsEnd"
   );
   const labelNode = (
     <div className="uif-toolsetting-label-docked-horizontal-empty">
-      {labelString}
+      {labelStringStart}{toolName ? toolName : toolPlaceholderName}{labelStringEnd}
```

Comment:
Could we work on **styling** and render an **icon** as suggested in the concept image?

Reply:
Need a note in `NextVersion.md` as well.

Reply:
I believe rendering the tool icon as an inactive picture in the tool settings is confusing.